frontend/src/router.js

```diff
@@ -47,117 +80,212 @@ const routerOptions = [
       requiresAuth: true,
-      requiresDataProducer: true
+      requiresDataProducer: true,
+      icon: mdiAccountMultiple,
+      title: 'Accounts',
+      menu: {
+        order: 6,
+        section: 'Entitlements'
+      },
+      dashboard: {
+        description:
+          'Managed accounts that are provisioned access to GCP resources through Datashare policies.',
+        order: 5
+      }
     }
   },
   {
-    path: '/editAccount',
-    name: 'editAccount',
-    component: 'EditAccount',
+    path: '/policies',
+    name: 'policies',
+    component: 'Policies',
     meta: {
       requiresAuth: true,
-      requiresDataProducer: true
+      requiresDataProducer: true,
+      icon: mdiBadgeAccount,
+      title: 'Policies',
+      menu: {
+        order: 7
+      },
+      dashboard: {
+        description:
+          'Policies allow data publishers to manage groupings of Datasets/Tables/PubSub Topics/Cloud Storage Buckets.',
+        order: 6
+      }
     }
   },
   {
-    path: '/policies',
-    name: 'policies',
-    component: 'Policies',
+    path: '/procurements',
+    name: 'procurements',
+    component: 'Procurements',
     meta: {
       requiresAuth: true,
-      requiresDataProducer: true
+      requiresDataProducer: true,
+      requiresMarketplaceIntegration: true,
+      icon: mdiShopping,
+      title: 'Procurement Requests',
+      menu: {
+        order: 8,
+        section: 'Marketplace'
+      },
+      dashboard: {
+        description:
+          'Manage procurements purchased through the GCP Marketplace.',
+        order: 7
+      }
     }
   },
   {
-    path: '/editPolicy',
-    name: 'editPolicy',
-    component: 'EditPolicy',
+    path: '/myProducts',
+    name: 'myProducts',
+    component: 'MyProducts',
     meta: {
       requiresAuth: true,
-      requiresDataProducer: true
+      requiresMarketplaceIntegration: true,
+      icon: mdiBriefcaseAccount,
+      title: 'My Products',
+      menu: {
+        order: 9
+      },
+      dashboard: {
+        description:
+          'View Datashare products that you have purchased through GCP Marketplace.',
+        order: 8
+      }
     }
   },
   {
-    path: '/procurements',
-    name: 'procurements',
-    component: 'Procurements',
+    path: '/admin',
+    name: 'admin',
+    component: 'Admin',
     meta: {
       requiresAuth: true,
       requiresDataProducer: true,
-      requiresMarketplaceIntegration: true
+      icon: mdiShieldKey,
+      title: 'Admin',
+      menu: {
+        order: 10,
+        section: 'Application',
+        subheader: 'Admin'
+      }
     }
   },
   {
-    path: '/activation',
-    name: 'activation',
-    component: 'Activation',
+    path: '/links',
+    name: 'links',
+    component: 'Links',
     meta: {
-      requiresAuth: true
+      icon: mdiLifebuoy,
+      title: 'Links',
+      menu: {
+        order: 11
+      },
+      dashboard: {
+        description: 'Find helpful Datashare links.',
+        order: 9
+      }
```

Comment:
I'm not sure if 'Links' is the right naming scheme for these assets. I.e. Would 'Documentation' be more appropriate vs hyper-'links'? @salsferrazza @swilliams11  thoughts?

Reply:
I was hesitant to put that card here. I don't think we need a docs/links card here. The other cards all have counts of something whereas this one doesn't. I think it can be removed. For a non-logged in user, we could re-direct them to the old welcome page which is 'links' currently.

Reply:
Also, for a logged in consumer, they would only see 1 card then, 'My Products', but they shouldn't ever really land up on the dashboard anyway. If they navigate through the GCP Marketplace, they'll be navigated directly to the my products page.

Reply:
Ok. Reverting back to the default welcome landing page and removing links makes sense.